src/cmake/modules/GUDHI_submodules.cmake

```diff
@@ -3,3 +3,9 @@ set(HERA_INTERNAL_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/ext/hera/include)
 set(HERA_INCLUDE_DIR ${HERA_INTERNAL_INCLUDE_DIR} CACHE PATH "Directory where one can find hera/{wasserstein.h,bottleneck.h}")
 # since everything is cleanly under include/hera/, there is no harm always including it
 include_directories(${HERA_INCLUDE_DIR})
+
+if (NOT EXISTS ${HERA_INCLUDE_DIR}/hera/wasserstein.h OR NOT EXISTS ${HERA_INCLUDE_DIR}/hera/bottleneck.h)
+  message(WARNING "${HERA_INCLUDE_DIR}/hera/{wasserstein.h,bottleneck.h} are not found.\n\
```

Comment:
Since the build is going to fail later anyway, shouldn't we have something stronger than a warning? If we error out immediately, the message is more likely to be seen than if it is lost in the middle pages of cmake+make stuff.

Reply:
I would let it as a warning, as if you say `cmake -DWITH_GUDHI_PYTHON=OFF`, hera is not required and it will compile fine